src/util/newconfig/ConfigDescription.hpp

```diff
@@ -136,8 +136,7 @@ struct ClioConfigDescription {
     static constexpr auto kCONFIG_DESCRIPTION = std::array{
         KV{.key = "database.type",
            .value = "Specifies the type of database to use for storing and retrieving data needed by the Clio server. "
-                    "Supported options are Cassandra and ScyllaDB. If you don't provide a value, the Clio server uses "
-                    "ScyllaDB by default."},
+                    "Only `Cassandra` is currently allowed."},
         KV{.key = "database.cassandra.contact_points",
```

Comment:
So here we are hardcoding `database.cassandra`. This may mean that we either need to copy the entire thing for `database.scylladb` or support it some other way by modifying the config framework.
@kuznetsss wdyt?